@kiva/kv-shop/src/shopError.ts

```diff
@@ -91,6 +91,23 @@ export function parseShopError(error: any) {
 		}, errorMessage);
 	}
 
+	if (
+		errorCode === 'upc.promoNotYetStarted'
+		|| errorCode === 'upc.promoHasEnded'
```

Comment:
please only look for the inclusion of `upc` within the `errorCode` and not the rest of the specific 2nd segment of code... There are several others and I don't think we should be copying those codes here to check against...

Reply:
oh I see you have it below, so you can just drop this first block...